python/src/otel/otel_sdk/nodeps-requirements.txt

```diff
@@ -1,31 +1,4 @@
-opentelemetry-instrumentation-aiohttp-client==0.47b0
-opentelemetry-util-http==0.47b0
-asgiref~=3.8
-opentelemetry-instrumentation-asgi==0.47b0
-opentelemetry-instrumentation-asyncpg==0.47b0
-opentelemetry-instrumentation-boto==0.47b0
-opentelemetry-instrumentation-boto3sqs==0.47b0
-opentelemetry-instrumentation-botocore==0.47b0
-opentelemetry-instrumentation-celery==0.47b0
-opentelemetry-instrumentation-dbapi==0.47b0
-opentelemetry-instrumentation-django==0.47b0
-opentelemetry-instrumentation-elasticsearch==0.47b0
-opentelemetry-instrumentation-fastapi==0.47b0
-opentelemetry-instrumentation-falcon==0.47b0
-opentelemetry-instrumentation-flask==0.47b0
-opentelemetry-instrumentation-grpc==0.47b0
-opentelemetry-instrumentation-jinja2==0.47b0
-opentelemetry-instrumentation-mysql==0.47b0
-opentelemetry-instrumentation-psycopg2==0.47b0
-opentelemetry-instrumentation-pymemcache==0.47b0
-opentelemetry-instrumentation-pymongo==0.47b0
-opentelemetry-instrumentation-pymysql==0.47b0
-opentelemetry-instrumentation-pyramid==0.47b0
-opentelemetry-instrumentation-redis==0.47b0
-opentelemetry-instrumentation-requests==0.47b0
-opentelemetry-instrumentation-sqlalchemy==0.47b0
-opentelemetry-instrumentation-sqlite3==0.47b0
-opentelemetry-instrumentation-starlette==0.47b0
-opentelemetry-instrumentation-tornado==0.47b0
-opentelemetry-instrumentation-wsgi==0.47b0
+# TODO: move these dependencies to requirements.txt when they stopped relying on a pinned version of
```

Comment:
Since [this](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/2773) is already merged we can make a new Python release and this won't be necessary.

Reply:
yes, this will work for v.0.48b0 (which is not released yet.).

Reply:
for the current release, 0.47.b0, it is still using the the pinned version.